keyboards/planck/keymaps/fsck/rules.mk

```diff
@@ -0,0 +1,5 @@
+ifndef QUANTUM_DIR
```

Comment:
Could you remove this (ifndef/include/endif) block? 

It's not needed, and is a holdover from the old build system.